src/lucky/routeable.cr

```diff
@@ -1,12 +1,25 @@
 module Lucky::Routeable
   {% for http_method in [:get, :put, :post, :delete] %}
+    # Define a route that responds to a {{ http_method.id.upcase }} request
+    #
+    # Use these methods if you are not using a restful route or a custom path.
```

Comment:
This makes it sound like you use this if you don't need a custom path. Maybe flip it? `Use these methods if you need a custom path or if you are not using a restful route`